packages/grid/_modules_/grid/hooks/features/rows/useGridRows.ts

```diff
@@ -231,6 +234,16 @@ export const useGridRows = (
 
   const updateRows = React.useCallback<GridRowApi['updateRows']>(
     (updates) => {
+      if (props.signature === GridSignature.DataGrid && updates.length > 1) {
+        // TODO: Add test with direct call to `apiRef.current.updateRows` in DataGrid after enabling the `apiRef` on the free plan.
+        throw new Error(
+          [
+            "MUI: You can't update several rows at once in `apiRef.current.updateRows` on the free plan",
+            'You need to upgrade to the DataGridPro component to unlock this feature.',
```

Comment:
```suggestion
            'You need to upgrade to the DataGridPro to unlock this feature.',
```

Reply:
We use the `DataGridPro component` on the other error messages (on the `chainPropTypes`)

Reply:
Ok, fine. +1 for keeping as it is.